packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -118,28 +141,30 @@ class _Compiler {
         while (compilationQueue.isNotEmpty) {
           final _CompilationRequest request = compilationQueue.first;
           printTrace('Compiling ${request.path}');
+          compiler ??= createCompiler();
+          suppressOutput = false;
           final String outputPath = await compiler.recompile(request.path,
             <String>[request.path],
             outputPath: outputDill.path,
           );
           // Copy output dill next to the source file.
-          final File kernelReadyToRun = await fs.file(outputPath).copy(
+          if (outputPath == null) {
```

Comment:
If you know in which cases we have `outputPath == null`, please add a comment here explaining this. It seems weird to request a compilation without using the result!

Reply:
Done.